lib/rules/indent.js

```diff
@@ -922,7 +922,11 @@ module.exports = {
 
             parameterParens.add(openingParen);
             parameterParens.add(closingParen);
-            offsets.setDesiredOffset(openingParen, sourceCode.getTokenBefore(openingParen), 0);
+
+            const offsetAfterToken = node.callee.type === "TaggedTemplateExpression" ? node.callee.quasi : openingParen;
```

Comment:
It looks like `offsets.setDesiredOffset()` is expecting a token. For consistency, should we do the following?

```suggestion
            const offsetAfterToken = node.callee.type === "TaggedTemplateExpression" ? sourceCode.getFirstToken(node.callee.quasi) : openingParen;
```

Reply:
@kaicataldo yes! thanks for review